kuma/wiki/models.py

```diff
@@ -1937,9 +1938,43 @@ class DocumentSpamAttempt(SpamAttempt):
         related_name='spam_attempts',
         null=True,
         blank=True,
-        verbose_name=ugettext('Document (optional)'),
+        verbose_name=_('Document (optional)'),
         on_delete=models.SET_NULL,
     )
+    data = models.TextField(
+        editable=False,
+        blank=True,
+        null=True,
+        verbose_name=_("Data submitted to Akismet")
+    )
+    reviewed = models.DateTimeField(
+        _('reviewed'),
+        blank=True,
+        null=True,
+    )
+
+    NEEDS_REVIEW = 0
+    HAM = 1
+    SPAM = 2
+    REVIEW_UNAVAILABLE = 3
+    REVIEW_CHOICES = (
+        (NEEDS_REVIEW, _('Needs Review')),
+        (HAM, _('Ham / False Positive')),
+        (SPAM, _('Confirmed as Spam')),
+        (REVIEW_UNAVAILABLE, _('Review Unavailable')),
+    )
```

Comment:
The pattern is double quotes, unless I notice I'm using them, and then I switch to single quotes. Single quotes aren't natural to me, and I'll be a lot more comfortable when [bug 1237977](https://bugzilla.mozilla.org/show_bug.cgi?id=1237977) is integrated.

Until then, I'll try to fix some of my double quotes and force push